plugin/cell-editor/cell.editor.view.js

```diff
@@ -0,0 +1,15 @@
+import {PluginView} from '../plugin.view';
+import {Event} from '@grid/core/infrastructure';
+
+export class CellEditorView extends PluginView {
+	constructor(model) {
+		super(...arguments);
```

Comment:
If you not planning to change number of constructor arguments, then we could use ``model`` instead